muikku-core-plugins/src/main/resources/fi/otavanopisto/muikku/plugins/transcriptofrecords/TranscriptofRecordsJsPlugin_fi.properties

```diff
@@ -1,72 +1,67 @@
-plugin.records.records.empty = Ei suorituksia
-plugin.records.records.title = Suoritukset
-plugin.records.errormessage.recordsLoadFailed = Suoritusten lataus ep�onnistui. Yrit� ladata sivu uudelleen.
-plugin.records.errormessage.workspaceLoadFailed = Kurssien lataus ep�onnistui. Yrit� ladata sivu uudelleen.
-plugin.records.errormessage.userWorkspaceLoadFailed = K�ytt�j�n ja ty�tilan lataus ep�onnistui. Yrit� ladata sivu uudelleen.
-plugin.records.hops.errormessage.hopsLoadFailed = HOPS-lataus ep�onnistui. Yrit� ladata sivu uudelleen.
-plugin.records.hops.errormessage.hopsUpdateFailed = HOPS-p�ivitys  ep�onnistui. Yrit� uudelleen.
-plugin.records.vops.errormessage.vopsLoadFailed = VOPS-lataus ep�onnistui. Yrit� ladata sivu uudelleen.
-
-plugin.records.files.title = Tiedostot
-plugin.records.files.empty = Ei tiedostoja
-plugin.records.workspaces.empty = Ei ty�tiloja
-plugin.records.workspace.evaluated = Arvioitu {0}
-
-plugin.records.summary.title = Opintojen yhteenveto
-plugin.records.summary.studyInfo = Opintojen tiedot
-plugin.records.summary.card.workspaces.title = Kurssisuoritukset
-plugin.records.summary.card.workspaces.stat.pre = Olet suorittanut yhteens�
-plugin.records.summary.card.workspaces.stat.post = abistatukseen vaadittavaa kurssia
-plugin.records.summary.card.activity.title = Aktiivisuus
-plugin.records.summary.card.activity.stat.pre = Olet kirjautunut yhteens�
-plugin.records.summary.card.activity.stat.post = kertaa viimeisen kuukauden aikana
-plugin.records.summary.card.tasks.title = Palautetut teht�v�t
-plugin.records.summary.card.tasks.stat.pre = Olet palauttanut yhteens�
-plugin.records.summary.card.tasks.stat.post = arvioitavaa teht�v�� opiskelujesi aikana
-plugin.records.summary.notifications.title = Ilmoitukset
-
-
+plugin.records.records.title = Records
```

Comment:
EN locale strings in FI properties file? 

Seems like there are mixed locale strings in FI and EN locale files in general regarding records view.

Reply:
Fucking 3 finger death paste